### PR TITLE
{devel}[GCCcore/10.2.0] PySide2 v5.14.2.3 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@SimonPinches please take a look at the style fixes identified in the unit tests.

**Comment by SimonPinches** [comment]
@verdurin I *think*  I've  fixed the style problems now.  (Although `--check-style` was already passing before...)

File: easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://download.qt.io/official_releases/QtForPython/%(namelower)s/%(name)s-%(version)s-src/']
+sources = ['pyside-setup-opensource-src-%(version)s.tar.xz']
+checksums = ['032de309505c78ba52c225aaf5fd717b8d87ae863aa7cee8ba58ba6141c59022']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('Sphinx', '3.5.2'),
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb:18]: @SimonPinches this version isn't available for `GCCcore/10.2.0`. There is `3.5.3` in `9.3.0`.

File: easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ('Clang', '11.0.1'),
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb:20]: @SimonPinches For `GCCcore`, you need to include `binutils` as a builddep. In this case, version `2.34`.

**Review by verdurin — DISMISSED** [review]

**Comment by SimonPinches** [comment]
@verdurin Seems I got things right this time :-)  Thanks for you help.

File: easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb (hunk 2 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb:20]: @verdurin I think it should be 2.35.

Comment (SimonPinches) [reply easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb:20]: Done :-)

File: easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb]: I've removed this dependency now.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/5f73968cc5a6f56719b8583699a569bf for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2620.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/8e064920618be39798598b9f7c52dd4e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**